.github/workflows/openstack-centos-7.yml

```diff
@@ -600,8 +599,8 @@ jobs:
             command_timeout: 1m
             script: |
               REPODIR=$(echo ${{ github.repository }} | cut -d / -f2)
-              /usr/local/cpanel/3rdparty/bin/prove -lvm /opt/${REPODIR}/t/integration/*.t
               /usr/local/cpanel/3rdparty/bin/prove -lvm /opt/${REPODIR}/t/integration/centos7-to-almalinux8/*.t
+              /usr/local/cpanel/3rdparty/bin/prove -lvm /opt/${REPODIR}/t/integration/*.t
```

Comment:
Because the OS specific tests are ensuring that the server actually upgraded. We can always flip them if/when more tests get added but for right now, it makes more sense in terms of the assertions for the OS specific tests to come first.